README.md

```diff
@@ -22,4 +22,12 @@ a pull request to the repository, you will be able to observe the results of
 the compliancy verification in your PR. Merge will be only possible if CI
 process is passed successfully.
 
+You can check the compliance with the linting rules by yourself with:
+
```

Comment:
i think it would be good to add requirements (e.g. ruby, i guess) and list as well the nodejs alternative used in the travis (given that at the end the ci will be run based on that)

Reply:
Hi,

Once I understood the real tool used I reverted this modification in c655ffa.

By default github updates the pull request if I push modification to the branch.
This should be true also for forced push. I can clean up the history and do a forced push if you prefer.